### ISSUE TITLE
validate on click issue

### ISSUE THREAD

**Issue by jasonwu513** [1000hz/bootstrap-validator#663]
Dear Sir:

I'm a newbie for bootstrap-validator , try to follow document and make a example but fail, no matter the validate pass or not , it will  alert fail once I click the input box , and even the input is correct , try to study #74 , but still not sure if I could use button click event , thanks in advnace.

`<!DOCTYPE html>
<html lang="">

<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Title Page</title>
    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet">
</head>

<body>

    <form role="form" data-toggle="validator" id="validator">
        <legend>Form title</legend>

        <div class="form-group">
            <label for="inputName" class="control-label">label</label>
            <input type="text" class="form-control" id="inputName" placeholder="Input field" required>
        </div>
        <div class="form-group">
            <label for="inputName2" class="control-label">label2</label>
            <input type="email" class="form-control" id="inputName2" placeholder="Input field" required>
        </div>

        <button type="submit" id="submitButton" form="validator" class="btn btn-primary">Submit</button>

    </form>

    
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
    
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/1000hz-bootstrap-validator/0.11.9/validator.min.js"></script>
    <script>
        $('#validator').validator().on('click', function (e) {
            e.preventDefault(); // prevent default
            if (e.isDefaultPrevented()) { // validate fail
                alert(' fail');
                return;
            } else { // validate
                alert('success');
            }

            alert('456');
        });
    </script>
</body>

</html>`
